client/src/styles/_variables.scss

```diff
@@ -1,6 +1,17 @@
-$base-spacing: 8px;
+$base-spacing: 0.5rem;
```

Comment:
yeah but how much pixels is that on a "normal" browser ?

Reply:
It's 8. The default font size on almost every browser is 16 pixels.

Reply:
👌